src/MoBi.Presentation/Nodes/TreeNodeFactory.cs

```diff
@@ -174,13 +175,7 @@ private void createAndAddNodeUnder(ITreeNode rootNode, IBuildingBlock buildingBl
 
       private ITreeNode createWithIcon(IBuildingBlock buildingBlock)
       {
-         var statusIcon = ApplicationIcons.GreenOverlayFor(buildingBlock.Icon);
-         // var statusIcon = buildingBlockInfo.BuildingBlockChanged
-         //    ? ApplicationIcons.RedOverlayFor(buildingBlock.Icon)
-         //    : ApplicationIcons.GreenOverlayFor(buildingBlock.Icon);
-
-         return CreateFor(buildingBlock)
-            .WithIcon(statusIcon);
+         return CreateFor(buildingBlock);
       }
```

Comment:
why not just call `CreateFor(buildingBlock)` instead of `createWithIcon(buildingBlock)`?
Then we can just remove this function